File: uikit/src/main/java/io/agora/chat/uikit/interfaces/OnItemLongClickListener.java
package io.agora.chat.uikit.interfaces;

import android.view.View;

/**
 * Item long click listener
 */
public interface OnItemLongClickListener {
    /**
     * Item long click
     * @param view
     * @param position
     */
    boolean onItemLongClick(View view, int position);
}

File: ease-im-kit/src/main/kotlin/com/hyphenate/easeui/interfaces/OnItemLongClickListener.kt
package com.hyphenate.easeui.interfaces

import android.view.View

/**
 * Item long click listener
 */
interface OnItemLongClickListener {
    /**
     * Item long click
     * @param view
     * @param position
     */
    fun onItemLongClick(view: View?, position: Int): Boolean
}